changelog.txt

```diff
@@ -1,4 +1,8 @@
 0.83
+ * BREAKING CHANGE: RealmResults.SORT_ORDER_ASCENDING and RealmResults.SORT_ORDER_DESCENDING constants
+   has been replaced by Sort.ASCENDING and Sort.DESCENDING enums.
```

Comment:
We used to not having line wraps in changelog. But be honest, i'd like to have it when line exceed some chars long (90 maybe?)